dlp/noxfile.py

```diff
@@ -118,6 +118,12 @@ def system(session):
     session.install("-e", "../test_utils/")
     session.install("-e", ".")
 
+    env = {
```

Comment:
I think you want something like what's in monitoring. That way the tests can work correctly in the CI for this repo and in the separate CI setup for VPCSC tests.

```
    # Additional setup for VPCSC system tests
    if os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC") != "true":
        # Unset PROJECT_ID, since VPCSC system tests expect this to be a project
        # within the VPCSC perimeter.
        env = {
            "PROJECT_ID": "",
            "GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT": os.environ.get("PROJECT_ID"),
        }
```

Reply:
fixed